src/aliases/store/models/alias.go

```diff
@@ -15,29 +22,64 @@ type Alias struct {
 	Value AliasValue
 }
 
-func AliasFromEntity(ent aliasent.Alias) Alias {
+func checkAliasValue(value aliasent.AliasValue) error {
+	for _, v := range value {
+		// we just want the decoding to happen without error
+		_, err := base64.StdEncoding.DecodeString(v)
+		if err != nil {
+			return errors.InvalidFormatError("alias value is incorrect: %v: %v", err, v)
+		}
+	}
+	return nil
+}
+
+func AliasFromEntity(ent aliasent.Alias) (alias Alias, err error) {
+	// check that the ent.Value only contains base64 string
+	err = checkAliasValue(ent.Value)
```

Comment:
It is not the responsibility of this function (or the data layer) to do this check. This check is business logic. Most of the time (if not always), constructors such as this should not return an error and `InvalidParameterErrors` come from the business layer.

Reply:
How do make sure the data is correctly formatted/coherent without error before entering the data storage?

Reply:
done